pages/linux/xman.md

```diff
@@ -0,0 +1,19 @@
+# xman
+
+> Manual page viewer for X Window System.
+
+- Start xman in three-button window:
+
+`xman`
+
+- Open helpfile other than default:
```

Comment:
Can you expand on this a bit ? What does a non-default helpfile mean ?

Reply:
Non-default helpfile stands for the man page (Used `man {{command}} > {{command}}.txt` to make these as a text file), which can be used, when pushing the `Manual page` button on the default window. (As I remember)

Reply:
So this is just the output of the man page stored to a file ? If so, can we say `Open manual page output stored in a given file` ?

Reply:
I guess yes